### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by kamba9** [jaykali/maskphish#122] [General]
Good evening, I just wanted to use your maskphish but, when I want to install it in termux, it won't let me, it asks me for username: and password: , if you know the username and password, could you share it with me? I need your invention for a job, please: (

**Comment by jaykali** [comment]
Ahhh, That's sounds like you did a typo. Please use the command `git clone https://github.com/jaykali/maskphish` command and make sure that you typed it correctly.
Thanks.
